<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurgeryDialog</class>
 <widget class="QDialog" name="SurgeryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Surgery</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="dateHorizontalLayout">
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="implantHorizontalLayout">
     <item>
      <widget class="QLabel" name="implantLabel">
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Implant: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="noneImplantRadioButton">
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">implantButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="leftImplantRadioButton">
       <property name="text">
        <string>Left</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">implantButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rightImplantRadioButton">
       <property name="text">
        <string>Right</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">implantButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bilatImplantRadioButton">
       <property name="text">
        <string>Bilateral</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">implantButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="injectionHorizontalLayout">
     <item>
      <widget class="QLabel" name="injectionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Injection: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="noneInjectionRadioButton">
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">injectionButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="leftInjectionRadioButton">
       <property name="text">
        <string>Left</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">injectionButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rightInjectionRadioButton">
       <property name="text">
        <string>Right</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">injectionButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bilatInjectionRadioButton">
       <property name="text">
        <string>Bilateral</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">injectionButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="procedureHorizontalLayout">
     <item>
      <widget class="QLabel" name="procedureLabel">
       <property name="text">
        <string>Procedure:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="procedureLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="anesthesiaHorizontalLayout">
     <item>
      <widget class="QLabel" name="anesthesiaLabel">
       <property name="text">
        <string>Anesthesia: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="anesthesiaLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="followUpHorizontalLayout">
     <item>
      <widget class="QLabel" name="followUpLabel">
       <property name="text">
        <string>Follow up care: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="followUpLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurgeryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurgeryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="implantButtonGroup"/>
  <buttongroup name="injectionButtonGroup"/>
 </buttongroups>
</ui>
